HISTORY.rst

```diff
@@ -1,7 +1,29 @@
 =======
 History
 =======
-3.41.0 (2012-08-05)
+
+3.42.0 (2021-08-19)
+-------------------
+Critical Changes
+
+* The ``github_release`` task now requires the ``tag_prefix`` option to be passed. (#2792)
+ 
+* The ``github_copy_subtree`` task now requires a ``tag_name`` option to be passed. (#2792)
+
+* The ``github_copy_subtree`` task option ``version`` has been deprecated (use the ``tag_name`` option instead). (#2792)
```

Comment:
Technically it's not required, and it probably makes sense to combine these, something like:

```suggestion
* In order to run the ``github_copy_subtree`` task for a specific package version, you must now use the ``tag_name`` option instead of the ``version`` option. Using the ``version`` option set to ``latest`` or ``latest_beta`` is deprecated; it's preferred to pass these values in the ``tag_name`` option instead. (#2792)
```